.gitignore

```diff
@@ -19,6 +19,7 @@ spec/reports
 /test_app/db/*.sqlite3
 /test_app/log/*.log
 /test_app/tmp/
+/test_app/Gemfile.lock
```

Comment:
I prefer to commit lock files in general. See https://bundler.io/blog/2018/01/17/making-gem-development-a-little-better.html for the rationale. So, if changing anything, I would remove `/Gemfile.lock` from this list and commit the root lock file instead.

Reply:
i need to track `activeadmin-mongoid/Gemfile.lock` in git, and `activeadmin-mongoid/test_app/Gemfile.lock` stays ignored like here, right?

Reply:
I would track both.